### PR TITLE
Update Deno Binary

### PR DESCRIPTION
**Pull request by habemuscode**

### CODE REVIEW

File: site/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   },
   "devDependencies": {
     "@vuepress/plugin-docsearch": "next",
-    "deno-bin": ">=1.31.1",
+    "deno-bin": "^1.34.3",
```
Comment (KnorpelSenf) [inline site/package.json:17]:
```suggestion
    "deno-bin": ">=1.34.3",
```

**Review by KnorpelSenf — APPROVED** [review]
Love love love

**Comment by roj1512** [comment]
Arghh wrong commit title again

**Comment by KnorpelSenf** [comment]
What are the commit titles you wish to see?

**Comment by roj1512** [comment]
Update deno-bin

**Pull request merged**